lib/ecto/adapters/sqlite3/connection.ex

```diff
@@ -800,7 +803,7 @@ defmodule Ecto.Adapters.SQLite3.Connection do
         # TODO: Should we have cell wise value support?
         #       Essentially ``?1 ?2 ?3`` instead of ``? ? ?``
         # {['?' | Integer.to_string(counter)], counter + 1}
-        {['?'], counter + 1}
```

Comment:
Sorry, my formatter apparently changed this?

Reply:
will remove

Reply:
Nah it's fine. The formatter is okay here. It's because of elixir 1.15

Reply:
ah, too late already removed it 😆

Reply:
Speaking of which, I need to bump the CI for elixir 1.15 and OTP 26 and drop 1.12 support.